test/versioned/koa/router-common.js

```diff
@@ -581,23 +578,20 @@ module.exports = (pkg) => {
         router.get('/:second', function terminalMiddleware(ctx) {
           ctx.body = 'this is a test'
         })
-        router.get('/second', function secondMiddleware(ctx) {
```

Comment:
I think this is where that test was introduced: https://github.com/newrelic/node-newrelic-koa/commit/4493aab0e2105b65c2b655dd80d02ef73e00ef42 but like i said before it's not actually hitting this handler